moto/ec2/utils.py

```diff
@@ -233,6 +233,10 @@ def random_launch_template_id() -> str:
     return random_id(prefix=EC2_RESOURCE_TO_PREFIX["launch-template"], size=17)
 
 
+def random_launch_template_name() -> str:
+    return f"LaunchTemplate_{random_resource_id(size=12)}"
```

Comment:
This is not _quite_ right, it is closer to what is produced by `cloudformation.utils.random_suffix` -- but importing `random_suffix` is circular.

This is probably close enough.